### PR TITLE
fix(profiling): disable 0 sample threads in selector

### PR DESCRIPTION
**Pull request by JonasBa**
Dont allow users to select 0 sample threads

### CODE REVIEW

File: static/app/components/profiling/threadSelector.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ function ThreadMenuSelector<OptionType extends GeneralSelectValue = GeneralSelec
         ? `tid (${profile.threadId}): ${profile.name}`
         : `tid (${profile.threadId})`,
       value: profile.threadId,
+      isDisabled: profile.samples.length === 0,
```
Comment (Zylphrex) [inline static/app/components/profiling/threadSelector.tsx:33]: What happens if every thread has 1 sample?

**Comment by JonasBa** [comment]
Oh lord, I rebased the wrong branch again. Too early

File: static/app/components/profiling/threadSelector.tsx [hunk 2]
```diff
@@ -30,6 +30,7 @@ function ThreadMenuSelector<OptionType extends GeneralSelectValue = GeneralSelec
         ? `tid (${profile.threadId}): ${profile.name}`
         : `tid (${profile.threadId})`,
       value: profile.threadId,
+      disabled: profile.samples.length === 0,
```
Comment (priscilawebdev) [inline static/app/components/profiling/threadSelector.tsx:33]: are we showing a tooltip or something to inform why this thread is disabled?

Comment (JonasBa) [reply static/app/components/profiling/threadSelector.tsx:33]: We display 0 samples in the dropdown and I assume it shouldn't be a problem for users to infer that that is why it's disabled. If there is confusion we can revisit later

**Review by priscilawebdev — APPROVED** [review]

**Pull request merged**